guides/source/debugging_rails_applications.md

```diff
@@ -337,7 +336,6 @@ Processing by ArticlesController#index as HTML
    10:     respond_to do |format|
    11:       format.html # index.html.erb
    12:       format.json { render json: @articles }
-
 (byebug)
```

Comment:
That change was a message update, and not just version upgrade. But yes, lets keep this, since you are running new commands on an updated version.